examples/js/bots/liquidity_bot.ts

```diff
@@ -0,0 +1,66 @@
+import { defaultClient } from "../client";
+import { ORDER_SIDE_ASK, ORDER_SIDE_BID, ORDER_TYPE_LIMIT, TestUser } from "../config";
+
+const base = "DIF";
+const quote = "BTC";
+const market = `${base}_${quote}`;
+const botUseFunds = 0.8; // Use 80% of funds
+const deviation = 1.002;
+const tiersAmount = 10;
```

Comment:
I'd use a waaay bigger number here. Play around with [this excel file](https://github.com/BitRAW/wiki/raw/master/assets/bots/LiquidityBotSampleData.ods) to get a feel for it.

Reply:
The max. orders in a batch right now is 40. I think in a first implementation we'll go with that but we can always go higher by using multiple batches